src/ppc64_cpu.c

```diff
@@ -371,22 +350,6 @@ static int get_cpu_info(void)
 	return 0;
 }
 
-static int is_smt_capable(void)
-{
-	struct stat sb;
-	char path[SYSFS_PATH_MAX];
-	int i;
-
-	for (i = 0; i < threads_in_system; i++) {
-		sprintf(path, SYSFS_CPUDIR"/smt_snooze_delay", i);
-		if (stat(path, &sb))
-			continue;
-		return 1;
-	}
-
-	return 0;
-}
-
```

Comment:
Removing this function is ok, good cleanup.  But, do we need alternative methods to check for SMT capabilities?

Reply:
If the system is not SMT capable, `threads_per_cpu` will be 1. `smt_state` will end up being 1, so you will see `SMT is off` (or `SMT=1` if you're in numeric mode).

If we wanted to preserve the same behavior as before this patch, I could change `is_smt_capable()` to return true if `threads_per_cpu > 1`

Reply:
I ended up keeping `is_smt_capable`